components/generic/RelatedChip.vue

```diff
@@ -28,8 +28,8 @@
         default: ''
       },
       linkGen: {
```

Comment:
As this is no longer a link _generating_ function, rename it to `linkTo`

Reply:
ok